src/test/kotlin/com/cultureamp/pg/RelationalDatabaseBookmarkStoreTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.cultureamp.pg
+
+import com.cultureamp.eventsourcing.Bookmark
+import com.cultureamp.eventsourcing.RelationalDatabaseBookmarkStore
+import io.kotest.core.spec.style.DescribeSpec
+import org.jetbrains.exposed.sql.Database
+import io.kotest.matchers.shouldBe
+import org.jetbrains.exposed.sql.SchemaUtils
+import org.jetbrains.exposed.sql.transactions.transaction
+
+class RelationalDatabaseBookmarkStoreTest : DescribeSpec({
+    val db = PgTestConfig.db ?: Database.connect(url = "jdbc:h2:mem:test;MODE=MySQL;DB_CLOSE_DELAY=-1;", driver = "org.h2.Driver")
```

Comment:
I am wondering if, since the only difference between this and the `com.cultureamp.eventsourcing` originals is this `PgTestConfig.db ?:` whether it might be better to remove the duplication and just fold this postgres run mode into those originals. What do you think?

Reply:
Right, that would be better.